frontend/src/app/main/ui/auth/recovery_request.cljs

```diff
@@ -102,3 +102,15 @@
                    :class (stl/css :go-back-link)
                    :data-testid "go-back-link"}
        (tr "labels.go-back")]]]))
+
+
+(mf/defc recovery-sent-page
+  [{:keys [email]}]
```

Comment:
missing `{::mf/props :obj}` metadata, it shuld be present on all new components.

Reply:
fixed